src/pages/job-offer-reject/overview/components/offered/offered.tsx

```diff
@@ -5,9 +5,8 @@ import css from './offered.module.scss';
 import { jobToApplicantListAdaptor } from './offered.services';
 import { OfferedProps } from './offered.types';
 import { endpoint } from '../../../../../core/endpoints';
```

Comment:
what is this import use for ?  it should be delete

Reply:
This component is related to job offer not chats, I didn't change anything except a wrong path which caused build error